cli/util/git_test.go

```diff
@@ -43,24 +44,52 @@ func TestGetRepoUrl(t *testing.T) {
 			wantErr: true,
 		},
 		{
-			name:   "simple w/ module sub directory",
-			repo:   "https://github.com/foo/bar",
-			subDir: "modules/bp1",
-			remote: defaultRemote,
-			want:   "https://github.com/foo/bar",
+			name:    "simple w/ module sub directory",
+			repo:    "https://github.com/foo/bar",
+			subDir:  "modules/bp1",
+			remote:  defaultRemote,
+			wantURL: "https://github.com/foo/bar",
+		},
+		{
+			name:    "simple w/ ssh remote",
+			repo:    "git@github.com:foo/bar.git",
+			remote:  defaultRemote,
+			wantURL: "https://github.com/foo/bar",
+		},
+		{
+			name:    "simple w/ module sub directory w/ ssh remote",
```

Comment:
Could we add a test case for gitlab or other provider just to assert we wont rewrite those

Reply:
Added.